### DISCUSSION TITLE
Paste image to chat like WA desktop in windows

### DISCUSSION THREAD

**Discussion by ihkaru** [eneshecan/whatsapp-for-linux#350] [Ideas]
Hello, i'm new in linux and currently using zorin os. I'm currently adapting to the new workflow of zorin os i'm using windows before. 

Is it possible to add "paste image to chat" feature like wa desktop in windows? 
it would be very nice update and helpful

**Comment by gui679** [comment]
Yesss!
Actually Web Whatsapp on any browser as Chrome or Firefox already does that
